### PR TITLE
[CGPROD-3102] Unit Tests tidy up

### PR DESCRIPTION
**Pull request by AdamXA**

### CODE REVIEW

File: src/components/shop/backgrounds.js [hunk 1]
```diff
@@ -41,7 +41,7 @@ export const initResizers = () => {
     resizers.set(Object, () => {});
 
     resizers.set(Phaser.GameObjects.Image, (scene, background, newSpec = {}) => {
-        const spec = { ...newSpec, ...defaultSpec };
+        const spec = { ...defaultSpec, ...newSpec };
```
Comment (AdamXA) [inline src/components/shop/backgrounds.js:44]: only code change, seems to have no impact 🤞

Comment (AnthonyGoodchild) [reply src/components/shop/backgrounds.js:44]: surprised it didn't cause issues the other way around

File: test/core/layout/text-elem.test.js [hunk 2]
```diff
@@ -2,7 +2,7 @@
  * @module components/shop
```
Comment (FostUK) [inline test/core/layout/text-elem.test.js:2]: Prob worth renaming this to text.test.js

**Review by FostUK — APPROVED** [review]
Nice job!!

**Pull request merged**